modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/Checkpointer.java

```diff
@@ -728,9 +718,9 @@ boolean isShutdownNow() {
      * <p>It helps when the cluster makes a checkpoint in the same time in every node.
      */
     long nextCheckpointInterval() {
-        long frequency = checkpointFrequencySupplier.getAsLong();
+        long frequency = checkpointConfig.frequency().value();
```

Comment:
I know I'm picking, but it would be better to use the same view to obtain both of these values, just in case

Reply:
I think we need to get the actual value from the configuration, so the view will not work for us.

Reply:
I'm sorry, I guess you didn't understand what I meant here. I implied the following:
```
var checkpointConfigView = checkpointConfig.value();

long frequency = checkpointConfig.frequency();
int deviation = checkpointConfig.frequencyDeviation();
```

Reply:
fix it